electron/extraction.js

```diff
@@ -0,0 +1,40 @@
+// The code below is a copy of cli.js from mcode-extraction-framework
+
+const { MCODEClient } = require('../node_modules/mcode-extraction-framework/src/client/MCODEClient');
+const logger = require('../node_modules/mcode-extraction-framework/src/helpers/logger');
+const { mcodeApp } = require('../node_modules/mcode-extraction-framework/src/cli/app');
```

Comment:
Because you installed these using NPM and because of how we set up the exports from the MEF, you don't need the file path to node_modules here. Instead, you can just import directly from `mcode-extraction-framework` like:

```
const { logger, mcodeApp, MCODEClient } = require('mcode-extraction-framework');
```

There are examples of this in the Base ICARE Client repository [here](https://github.com/mcode/base-icare-extraction-client/blob/main/src/app.js#L5-L8) in case you are curious to see it done in other places!